pulp_smash/constants.py

```diff
@@ -655,6 +655,12 @@
 )
 """The URL to an RPM with non-UTF-8 metadata in its header."""
 
+RPM_WITH_VENDOR_FEED_URL = urljoin(
+    PULP_FIXTURES_BASE_URL,
+    'rpm-with-vendor/'
+)
+"""A repository whose primary.xml file has an vendor section."""
+
 RPM_WITH_VENDOR_URL = urljoin(
     PULP_FIXTURES_BASE_URL,
```

Comment:
Can this constant be derived from `RPM_WITH_VENDOR_FEED_URL` instead of `PULP_FIXTURES_BASE_URL`?

Reply:
Done.